crates/re_viewer/src/ui/data_ui/image.rs

```diff
@@ -24,7 +24,22 @@ impl DataUi for Tensor {
         verbosity: crate::ui::UiVerbosity,
         _query: &re_arrow_store::LatestAtQuery,
     ) {
-        let tensor_view = ctx.cache.image.get_view(self, ctx.render_ctx);
+        let decoded = ctx
+            .cache
+            .decode
+            .try_decode_tensor_if_necessary(self.clone());
+
+        let tensor_view = match &decoded {
```

Comment:
this now always copies the tensor if I'm not mistaken. Previously, this would only get a view into the tensor cache

Reply:
This now uses an Arrow Buffer which is cheap to clone.